sphinx/builders/html/__init__.py

```diff
@@ -315,8 +315,9 @@ def init_js_files(self) -> None:
         self.script_files = []
         self.add_js_file('documentation_options.js', id="documentation_options",
                          data_url_root='', priority=200)
-        self.add_js_file('jquery.js', priority=200)
-        self.add_js_file('underscore.js', priority=200)
+        self.add_js_file('jquery.js', priority=200)  # Remove in Sphinx 6.0
+        self.add_js_file('underscore.js', priority=200)  # Remove in Sphinx 6.0
+        self.add_js_file('_compat.js', priority=200)  # jQuery compatability shim.
```

Comment:
nit: Mention `RemovedInSphinx60Warning` in a comment around this, since it is likely what someone would search for when determining what all to remove as part of preparing for Sphinx 6.0.